tutorials/appstudio-onboarding/appstudio-onboarding.md

```diff
@@ -58,11 +62,11 @@ Go to <https://account.hanatrial.ondemand.com> and log in to your SAP Cloud Plat
 
 [ACCORDION-BEGIN [Step: 3](Add SAP Business Application Studio subscription to subaccount)]
 
-In this step, you will add the SAP Business Application Studio subscription to a subaccount. Depending on when you created the SAP Cloud Platform account, this subscription might already be added in the subaccount.
+In this step, you will add the SAP Business Application Studio subscription to a subaccount. Depending on when you created the SAP BTP trial account, this subscription might already be added in the subaccount.
 
 1. Click **Enter Your Trial Account** to access the Cloud Foundry environment.
 
-    !![Access SAP Cloud Platform Trial](2020-08 SCP Access Trial_.jpg)
+    !![Access SAP BTP Trial](2020-08 SCP Access Trial_.jpg)
```

Comment:
Please follow the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtvf3pb64) when naming your images.